### ISSUE TITLE
Should we enable merge-queue for this repository?

### ISSUE THREAD

**Issue by CYBAI** [servo/internal-wpt-dashboard#12]
While looking at PRs in this repo, I noticed that we haven't enabled merge-queue for this repo yet.
Because this repo is quite new, it doesn't use bors but I wonder maybe it's still good to enable merge-queue for this repo, what do you think?

**Comment by mukilan** [comment]
Good point. I'm not sure if the current main workflow is compatible with Merge Queues, which requires it to execute on both pull request and merge_group events. I'll look into this. Also, most of the commits are directly pushed to main branch by the CI job, so I'll have to check if that can coexist with the merge queue branch protections.
